CHANGELOG.md

```diff
@@ -6,10 +6,30 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+- **Breaking change**: Changed API of many sheet-related methods to take sheetId instead of sheetName as an argument. (#645)
+- **Breaking change**: Removed support for matrix formulas (`{=FORMULA}`) notation. Engine now supports formulas returning array of values (instead of only scalars). (#652)
+- **Breaking change**: Removed numeric matrix detection along with matrixDetection and matrixDetectionThreshold config options. (#669)
+- Changed SWITCH function so it takes array as its first argument.
+
```

Comment:
Are those really going to be in `unreleased`? Weren't those were already part of the `develop` that went into `master` branch?

Reply:
The last two releases were hotfixes based on `master`.